pom.xml

```diff
@@ -41,13 +42,9 @@
       <artifactId>plexus-utils</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-container-default</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-component-annotations</artifactId>
-      <optional>true</optional>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
```

Comment:
Should this be optional or provided?

Reply:
Optional would make it impossible for sisu to discover components at runtime (if CP would not have it). So imo, not optional. Am on edge here, but afair in 99% of cases is not optional.